sauna/main.py

```diff
@@ -70,3 +70,6 @@ def main():
         sauna_instance.launch()
 
     logging.shutdown()
+
+if __name__ == '__main__':
```

Comment:
I believe this is not necessary. Today's good practice seems to be using entrypoints in `setup.py`.
When installed with `pip install -e .` you can run Sauna the same way it will be run when installed.

See for instance http://click.pocoo.org/5/quickstart/#switching-to-setuptools